File: 1. Learn the Basics/1. Pattern/C++/14.cpp
/*

A 
A B 
A B C 
A B C D 
A B C D E  

*/

#include <bits/stdc++.h>
using namespace std;

void nLetterTriangle(int n) {
    // Write your code here
    for(int i=0;i<n;i++){
        int letter = 65;
        for(int j=0;j<=i;j++){
            cout << (char)letter << " ";
            letter++;
        }
        cout << endl;
    }
}
int main() {
	// your code goes here
	int n;
	cin >> n;
	nLetterTriangle(n);
	return 0;
}

File: 1. Learn the Basics/1. Pattern/Java/14.java
/* package codechef; // don't place package name! */

/*

A 
A B 
A B C 
A B C D 
A B C D E 

*/
import java.util.*;
import java.lang.*;
import java.io.*;


/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
    public void nLetterTriangle(int n) {
        // Write your code here
        
        for(int i=0;i<n;i++){
            int letter = 65;
            for(int j=0;j<=i;j++){
                System.out.print((char)letter+" ");
                letter++;
            }
            System.out.println();
        }
    }
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Codechef obj = new Codechef();
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		obj.nLetterTriangle(n);
	}
}
